resources/views/community/components/news/carousel-item.blade.php

```diff
@@ -5,8 +5,8 @@ class="relative box-content flex w-full flex-none snap-start min-h-[270px]"
     {{ $index === 0 ? 'id=first-news-item' : '' }}
     {{ $index === $totalCount - 1 ? 'id=last-news-item' : '' }}
 >
-    <div 
-        class="z-10 py-6 px-8 absolute top-0 left-0 w-full h-full"
```

Comment:
Not sure if the z-index was necessary because of the menu dropdowns before but leaving those im place put the news items in front of the dropdowns.

Reply:
Without this z-index value, the text appears behind the images.
This can be directly observed by moving the mouse over the author's name. The tooltip no longer appears because the text is not being hovered over.

Reply:
I see! I'll revisit that then

Reply:
Changing the render order of items seems to do the trick for me without z-index 🤔 
https://github.com/RetroAchievements/RAWeb/pull/1776/commits/32c6f8686940e846825bfcac3593c6a332b39913

Reply:
Confirmed, news carousel looks good now